### PR TITLE
MAP-1

### PR DESCRIPTION
**Pull request by suniala**

### CODE REVIEW

File: api-1.9/pom.xml [hunk 1]
```diff
@@ -35,21 +36,40 @@
...
+			<version>${openmrs.1.9.api.version}</version>
+			<!--$NO-MVN-MAN-VER$-->
 			<type>pom</type>
 		</dependency>
 	</dependencies>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>com.googlecode.maven-java-formatter-plugin</groupId>
```
Comment (rkorytkowski) [inline api-1.9/pom.xml]: I'm surprised you need to add it to a pom of every submodule. I think it should be enough to have it in the parent pom only.

File: api-common/src/main/java/org/openmrs/module/metadatamapping/api/adapter/ConceptAdapterPre19.java [hunk 2]
```diff
@@ -1,92 +1,92 @@
...
+import org.openmrs.ConceptMap;
+import org.openmrs.ConceptSource;
+import org.openmrs.api.context.Context;
+import org.springframework.stereotype.Component;
+
+/**
+ * Provides support for the Concept class pre OpenMRS 1.9.
+ */
+@Component("metadatamapping.ConceptAdapterPre19")
+public class ConceptAdapterPre19 implements ConceptAdapter {
```
Comment (rkorytkowski) [inline api-common/src/main/java/org/openmrs/module/metadatamapping/api/adapter/ConceptAdapterPre19.java]: We could actually drop support for pre 1.9 at this point and support only 1.9+

File: api-common/src/main/resources/sqldiff.xml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE sqldiff PUBLIC "-//OpenMRS//DTD OpenMRS SQL Diff Config 1.0//EN" "http://resources.openmrs.org/doctype/sqldiff-1.0.dtd">
+<!DOCTYPE sqldiff PUBLIC "-//OpenMRS//DTD OpenMRS SQL Diff Config 1.0//EN"
```
Comment (rkorytkowski) [inline api-common/src/main/resources/sqldiff.xml]: You can get rid of sqldiff.xml. We will support 1.9+.

File: omod/src/main/resources/config.xml [hunk 4]
```diff
@@ -10,13 +10,13 @@
...
 	<activator>@MODULE_PACKAGE@.MetadataMappingActivator</activator>
-
+	
 	<updateURL>https://modules.openmrs.org/modules/download/@MODULE_ID@/update.rdf</updateURL>
-
+	
 	<!-- /Metadata Sharing Properties -->
-
+	
 	<require_version>1.6</require_version>
```
Comment (rkorytkowski) [inline omod/src/main/resources/config.xml]: Maybe it's about time to require 1.9

**Comment by rkorytkowski** [comment]
Please feel free to create an issue to drop support for OpenMRS older than 1.9

File: api-1.9/pom.xml (hunk 1 above)
Comment (suniala) [inline api-1.9/pom.xml]: At first, I did try just adding it to the parent pom but it did not work. Of course it is possible I had something else missing from the configuration at that time.

**Pull request merged**